explorer/app/components/Detail/components/GeneratedMatricesTables/generatedMatricesTables.tsx

```diff
@@ -0,0 +1,59 @@
+import { GridPaper } from "@clevercanary/data-explorer-ui/lib/components/common/Paper/paper.styles";
+import { Table } from "@clevercanary/data-explorer-ui/lib/components/Detail/components/Table/table";
+import { Divider } from "@mui/material";
+import { ColumnDef } from "@tanstack/react-table";
+import React, { Fragment } from "react";
+import { GenusSpecies } from "../../../../viewModelBuilders/azul/common/constants";
+import { ProjectMatrixTableView } from "../../../../viewModelBuilders/azul/common/entities";
+import { Alert, SectionTitle } from "./generatedMatricesTables.styles";
+
+interface GeneratedMatricesTablesProps<T extends object> {
+  columns: ColumnDef<T>[];
+  gridTemplateColumns: string;
+  projectMatrixViewsBySpecies: ProjectMatrixTableView[];
+}
+
+export const GeneratedMatricesTables = <T extends object>({
+  columns,
+  gridTemplateColumns,
+  projectMatrixViewsBySpecies,
+}: GeneratedMatricesTablesProps<T>): JSX.Element => {
+  return (
+    <>
+      {projectMatrixViewsBySpecies.length > 0 ? (
+        projectMatrixViewsBySpecies.map(
+          ({ projectMatrixViews, species }, i) => {
+            const title = getSpeciesDisplayText(species);
+            const items = projectMatrixViews as T[]; // TODO review type assertion.
```

Comment:
@NoopDog note the type assertion on line `27` - we need `projectMatrixViews` (an array of `projectMatrixView`) to be typed `T[]` as per table items prop.